README.md

```diff
@@ -6,7 +6,7 @@
 
 This action…
 
-- Installs Snapcraft on macOS/Ubuntu
+- Installs Snapcraft on macOS / Ubuntu 18.04+
```

Comment:
```suggestion
- Installs Snapcraft on macOS/Ubuntu 18.04+
```

Reply:
intriguing since I don't see any particular reason 16.04 should fail (based on https://github.com/actions/virtual-environments/releases)

Reply:
I made it on purpose to make it clearer that the version number doesn't belong to macOS. 😀  But both are fine with me...

Reply:
well then...

```suggestion
- Installs Snapcraft on Ubuntu 18.04+/macOS
```